### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by rhattersley**

### CODE REVIEW

**Comment by rhattersley** [comment]
Ping @SciTools/iris-grib-devs

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+1. Remember to add appropriate documentation and tests to supplement any new or changed functionality.
+
+
+Submitting changes
+------------------
+
+1. Read and sign the Contributor Licence Agreement (CLA).
+ - See our [governance page](http://scitools.org.uk/governance.html)
+   for the CLA and what to do with it.
+1. Push your branch to your fork of iris-grib.
```
Comment (pelson) [inline CONTRIBUTING.md]: The markdown numbering is silly.

Comment (ajdawson) [inline CONTRIBUTING.md]: Agreed, wouldn't it render equally well if we used, say, the actual number you want... A lot of people will read this a plain text and it looks weird like this.

Comment (rhattersley) [inline CONTRIBUTING.md]: Fixed in #4

**Pull request merged**